crates/pathfinder/src/sequencer.rs

```diff
@@ -648,6 +699,14 @@ mod tests {
         }
     }
 
+    impl std::fmt::Display for crate::core::ClassHash {
```

Comment:
I would like to refactor the StarkHash to actually implement `LowerHex` and `UpperHex` using `as_hex_str` and hence dropping the leading zeros (in a separate PR ofc).
As a side effect these local `Display` impls would be dropped.
:+1: or :-1: if you agree/disagree

Reply:
So in the end `<StarkHash as Display>::fmt` would give you full hex and and x/X for possibly shorter? That would make sense to me.

Related: I've longed for transparent display for these starkhash wrappers for quite some time, lowerhex and upperhex would be similarly easy to macro_rules! implement.

Reply:
:+1:

Reply:
Oh, this is a very good idea to add some useful impls for those newtypes using a macro.